apps/prairielearn/src/ee/pages/instructorAiGenerateJobReview/instructorAiGenerateJobReview.sql

```diff
@@ -0,0 +1,11 @@
+-- BLOCK select_generation_sequence_by_course
+SELECT
+  j.*
+FROM
+  jobs AS j
+WHERE
+  j.course_id IS NOT DISTINCT FROM $course_id
```

Comment:
Why not this?

```suggestion
  j.course_id = $course_id
```

Reply:
saw course ids being compared like this elsewhere, but will change over

Reply:
changed in 3c8bafd1470728a8c79b488213de1dfae0aed79e.